README.md

```diff
@@ -0,0 +1,42 @@
+# MEmilio-simulations
+This Repository contains simulations using the library [MEmilio](https://github.com/SciCompMod/memilio).
+In this Repo, the simulations are connected with the MEmilio version used to create the simulations. 
+This way, the outputs should be easy to recreate. 
+All files used to create outputs used e.g. for paper simulations, can be included. 
+The scripts do not necessarily have to be portable (e.g. some shellscripts), but they worked locally. Please be aware of tis when using the files.
+
+## Configuring using CMake
+
+To configure with default options (no simulations will be built):
+```bash
+mkdir build && cd build
+cmake ..
+```
+
+You can specify which simulation folders to build by specifying options via `cmake .. -D<OPTION>=<VALUE>`.
+All options can be set to ON or OFF and the default value is always OFF. Please check the README of the subdirectories for details about the simulations.
+The following options can be set:
+- `BUILD_2020_npis_sarscov2_wildtype_germany`
+- `BUILD_2021_vaccination_sarscov2_delta_germany`
+- `BUILD_munich_graph_sim`
+- `BUILD_2024_Ploetzke_Linear_Chain_Trick`.
+
+A build folder with the correct MEmilio version will be created in each subdirectory. 
+You can run a simulation with e.g.:
+```bash
+./2020_npis_sarscov2_wildtype_germany/build/2020_npis_wildtype
+```
```

Comment:
If you are still in the build directory, this wont work. Maybe add 
```
cd .. # if you are still in the build directory
```
or just make the path start with ".." instead, or use `cmake -B build .. -D<...>` instead of `mkdir build && cd build`. I personally like the last one most, though using mkdir is more common.

Reply:
I chose the option to start the path with ../